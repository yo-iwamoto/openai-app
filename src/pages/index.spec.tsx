import Page from './index.page';
import { render } from '@testing-library/react';

describe('[page] /', () => {
  it('snapshot', () => {
    const { asFragment } = render(<Page />);

    expect(asFragment()).toMatchSnapshot();
  });
});
