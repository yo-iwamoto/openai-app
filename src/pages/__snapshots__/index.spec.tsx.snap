// Vitest Snapshot v1

exports[`[page] / > snapshot 1`] = `
<DocumentFragment>
  <h1>
    Hello world
  </h1>
</DocumentFragment>
`;
